test/extensions/filters/http/lua/wrappers_test.cc

```diff
@@ -68,6 +68,50 @@ TEST_F(LuaHeaderMapWrapperTest, Methods) {
   start("callMe");
 }
 
+// Get the value size of a certain header with multiple values.
+TEST_F(LuaHeaderMapWrapperTest, GetValueSize) {
```

Comment:
Can we have tests when the index is wrong? e.g. -1 and bigger than the size of the values.

Reply:
We already have a test for the bigger size. I also added one for `-1`.